microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/MetricsTest.java

```diff
@@ -55,15 +55,12 @@ public static WebArchive createDeployment() {
 
     @Before
     public void prepare() {
+        String metricsURL = "http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + "/metrics";
         jsonMetricsRequest = given()
-                .baseUri("http://" + managementClient.getMgmtAddress())
-                .port(managementClient.getMgmtPort())
-                .basePath("metrics")
+                .baseUri(metricsURL)
```

Comment:
IMO we should have same approach to use rest assured API across all modules. `baseUri` with fully-qualified address vs `baseUri` + `port` + `basePath`. Can you create an issue for this?

Reply:
here we go https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/140